### ISSUE TITLE
Required positional argument: 'on_delete'

### ISSUE THREAD

**Issue by md1023** [synw/django-mptt-graph#2]
Django 2.0 throws error when I add mptt_graph into INSTALLED_APPS.
```
  File "/usr/local/lib/python3.6/site-packages/mptt_graph/models.py", line 22, in <module>
    class TreeNode(MPTTModel):
  File "/usr/local/lib/python3.6/site-packages/mptt_graph/models.py", line 24, in TreeNode
    parent = TreeForeignKey('self', null=True, blank=True, related_name=u'children', verbose_name=_(u'Parent node'))
TypeError: __init__() missing 1 required positional argument: 'on_delete'
```

**Comment by synw** [comment]
I see: this is because Django 2 requires the `on_delete` parameter to be set on foreign keys. I will fix this asap. I'm not sure about the urls compatibility though, not using Django 2 yet

Thanks for reporting

**Comment by synw** [comment]
I fixed this and made a 0.3 release that works with Django 2. Let me know if you encounter any other problem
